apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/dgs/DGSServiceSearchTest.java

```diff
@@ -49,12 +53,22 @@
             DGSTagTable.class,
             DGSPurchaseTable.class,
             DGSServiceImpl.class,
-            DerivedDbTablesRegistryImpl.class)
+            DerivedDbTablesRegistryImpl.class,
```

Comment:
Create account_service independently or mock classes which are not required in this test

Reply:
partially done, this test depends on the account service/table behavior.